pyproject.toml

```diff
@@ -80,6 +80,7 @@ youtube-transcript-api = "^0.6.2"
 posthog = "^3.5.0"
 ruff = "0.5.1"
 django-silk = "^5.1.0"
+timefhuman = "^0.0.5"
```

Comment:
This was last updated in 2018, which is concerning.  I think [dateparser](https://pypi.org/project/dateparser/) might be a better choice, it is more up to date (11/2023) and seems more popular.

Reply:
good point. swapped it out.